tutorials/io/plot_30_reading_fnirs_data.py

```diff
@@ -60,17 +61,135 @@
+# Next, we will load the example csv file.
+# The metadata must be specified manually as the csv file does not contain
+# information about channel names, types, sample rate etc.
+
+data = pd.read_csv('fnirs.csv')
+
+# In MNE the naming of channels MUST follow this structure of
+# `S#_D# type` or `S#_D# wavelength`, where # is replaced
+# by the appropriate source and detector number, type is
+# either hbo or hbr, and wavelength is specified in nm.
+
+ch_names = ['D1_S1 hbo', 'D1_S1 hbr', 'D1_S2 hbo', 'D1_S2 hbr',
+            'D1_S3 hbo', 'D1_S3 hbr', 'D1_S4 hbo', 'D1_S4 hbr',
+            'D2_S5 hbo', 'D2_S5 hbr', 'D2_S6 hbo', 'D2_S6 hbr',
+            'D2_S7 hbo', 'D2_S7 hbr', 'D2_S8 hbo', 'D2_S8 hbr']
+ch_types = ['hbo', 'hbr', 'hbo', 'hbr',
+            'hbo', 'hbr', 'hbo', 'hbr',
+            'hbo', 'hbr', 'hbo', 'hbr',
+            'hbo', 'hbr', 'hbo', 'hbr']
+sfreq = 10.  # Hz
+
+
+###############################################################################
+# Finally, the data can be converted in to an MNE data structure.
+# The metadata above is used to create an :class:`mne.Info` data structure,
+# and this is combined with the data to create
+# an MNE :class:`~mne.io.Raw` object. For more details on the info structure
+# see :ref:`tut-info-class`, and for additional details on how continuous
+# data is stored in MNE see :ref:`tut-raw-class`.
+# For a more extensive description of how to create MNE data structures from
+# raw array data see :ref:`tut_creating_data_structures`.
+
+info = mne.create_info(ch_names=ch_names, ch_types=ch_types, sfreq=sfreq)
+raw = mne.io.RawArray(data, info, verbose=True)
+
+
+###############################################################################
+# Applying standard sensor locations to imported data
+# ---------------------------------------------------
+#
+# Having information about optode locations may assist in your analysis.
+# Beyond the general benefits this provides
+# (e.g. creating regions of interest, etc),
+# this is particularly important for fNIRS as information about the
+# distance between optodes is required to convert the optical density data
+# in to an estimate of the haemoglobin concentrations.
+# MNE provides methods to load standard sensor configurations (montages) from
+# some vendors, and this is demonstrated below.
+# Some handy tutorials for understanding sensor locations, coordinate systems,
+# and how to store and view this information in MNE are:
+# :ref:`tut-sensor-locations`, :ref:`plot_source_alignment`, and
+# :ref:`ex-eeg-scalp`.
+#
+# Below is an example of how to load the optode positions for an Artinis
+# OctaMon device. However, many fNIRS researchers use custom optode montages,
+# in this case you can generate your own .elc file (see `example file
+# <https://github.com/mne-tools/mne-python/blob/main/mne/channels/data
+# /montages/standard_1020.elc>`_) and load that instead.
+
+raw.set_montage('artinis-octamon')
```

Comment:
@rob-luke, I removed
```python
# To load a custom montage use:
# raw.set_montage('/path/to/custom/montage.elc')
```
as it is not possible to set a montage from a custom file (even with other modalities than fNIRS).
Maybe that should be a separate PR if that's really needed.

Reply:
> Oh actually it kinda is

So maybe we can add in a small comment like before that would be something like `raw.set_montage(make_dig_montage("path/to/file))`, or whatever the correct is.

Reply:
I'm afraid it'll add a lot of comments as it does not work directly from an elc file with this function, it requires arrays, so there is some manual parsing of the elc to do in that case (*_read_elc* is a private method). Should we just point to the *make_dig_montage* doc maybe?

Reply:
Lets skip for now 👍 leave it out.